### DISCUSSION TITLE
Resource collections, and pagination

### DISCUSSION THREAD

**Discussion by ghost** [dedoc/scramble#56] [Q&A] [answered]
Add more visibility in form of example: 

data (array)
And then how the objects look.
Instead of this:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/102802845/195078038-ac758e60-5c3c-45a2-accb-b834136fa404.png">

Also pagination is not shown in the docs.

**Comment by romalytvynenko** [comment, marked as answer]
Hey @Jesper-FIltenborg-Dwarf 

In `0.6.0` the issue with incorrect object has been fixed.

To add pagination info to docs, you need to manually type hint it with `LengthAwarePaginator`: https://scramble.dedoc.co/usage/response#lengthawarepaginator-response

Let me know if you have other questions!
